container-storage-setup.spec

```diff
@@ -15,10 +15,7 @@ URL:            http://github.com/projectatomic/container-storage-setup/
 Source0: %{git0}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 BuildArch: noarch
 
-BuildRequires:  pkgconfig(systemd)
-
 Requires:       lvm2
-Requires:       systemd-units
```

Comment:
@lsm5 Ok, I will buy it for now. If somebody takes this spec and tries to use it on a non-systemd distro, i think they would run into issues.